### ISSUE TITLE
[REQUEST] Swap network download/upload graphs

### ISSUE THREAD

**Issue by Sheridan** [aristocratos/bpytop#383]
Or make option for this.
Because **down**load and **up**load

**Comment by NavigationHazard** [comment]
Swap them to what? Can you please specify this a bit more or add suggestion picture. :)

**Comment by Sheridan** [comment]
Network download and upload
![image](https://user-images.githubusercontent.com/728591/164736149-3ada7ee7-176e-41ad-862a-60abcd5c9cf3.png)

**Comment by Sheridan** [comment]
Network download and upload

![image](https://user-images.githubusercontent.com/728591/164736488-5fe34e8e-a7df-48c0-832c-3c91f869302e.png)

**Comment by abesnier** [comment]
I concur, I find it really disturbing to have the up link down, and the down link up.

There are three places to change if you want to do it yourself (line numbers as per [current master](https://github.com/aristocratos/bpytop/blob/master/bpytop.py):)

[Line 2372](https://github.com/aristocratos/bpytop/blob/master/bpytop.py#L2372)
`{create_box(x=cls.box_x, y=cls.box_y, width=cls.box_width, height=cls.box_height, line_color=THEME.div_line, fill=False, title="Download", title2="Upload")}' `becomes 	`{create_box(x=cls.box_x, y=cls.box_y, width=cls.box_width, height=cls.box_height, line_color=THEME.div_line, fill=False, title="Upload", title2="Download")}'`

[Line 2409](https://github.com/aristocratos/bpytop/blob/master/bpytop.py#L2409)
`for direction in ["download", "upload"]:` becomes `for direction in ["upload", "download"]:`

[Lines 2430/2431](https://github.com/aristocratos/bpytop/blob/master/bpytop.py#L2430) 
```
out += (f'{Mv.to(y, x)}{THEME.graph_text(net.sync_string if CONFIG.net_sync else net.strings[net.nic]["download"]["graph_top"])}'
				f'{Mv.to(y+h-1, x)}{THEME.graph_text(net.sync_string if CONFIG.net_sync else net.strings[net.nic]["upload"]["graph_top"])}')
```
become
```
out += (f'{Mv.to(y, x)}{THEME.graph_text(net.sync_string if CONFIG.net_sync else net.strings[net.nic]["upload"]["graph_top"])}'
				f'{Mv.to(y+h-1, x)}{THEME.graph_text(net.sync_string if CONFIG.net_sync else net.strings[net.nic]["download"]["graph_top"])}')
```

That's a permanent change though, not a config option, and will be reverted when you update BPYTOP.

**Comment by Crescendo-BLYAT** [comment]
Yeah, current implementation is confusing.... Up is download, down is upload.
